libsolidity/formal/Why3Translator.cpp

```diff
@@ -757,6 +757,11 @@ bool Why3Translator::visit(Literal const& _literal)
 	return false;
 }
 
+bool Why3Translator::visit(PragmaDirective const&)
+{
+	return false;
```

Comment:
The idea is that pragma statements are identified by their first identifier, so `pragma solidity` should always be a version pragma.